fdbserver/workloads/IDDTxnProcessorApiCorrectness.actor.cpp

```diff
@@ -246,6 +318,8 @@ struct IDDTxnProcessorApiWorkload : TestWorkload {
 		                   .get();
 
 		verifyInitDataEqual(self->realInitDD, mockInitData);
+		TraceEvent(SevDebug, relocateShardInterval.end(), relocateShardInterval.pairID);
+		self->mock->setupMockGlobalState(self->realInitDD); // in case SS remove or recruit
```

Comment:
I'm not sure I understand the comment for this line. Why do we need to call `setupMockGlobalState` here?

Reply:
Because sometimes the simulator will generate a scenario when after the first `setupMockGlobalState` call, there is a new storage server join the cluster, there's no way for mock DD to know the new storage server without call setupMockGlobalState again.

Reply:
Thanks, that makes sense. Can you expand the comment to say this?